File: solutions/1768/java/Solution.java
// 1768. Merge Strings Alternately
class Solution
{
	public String mergeAlternately(String word1, String word2)
	{
		StringBuilder result = new StringBuilder();

		for (int i = 0;; ++i)
		{
			boolean isComplete = true;

			if (i < word1.length())
			{
				result.append(word1.charAt(i));

				isComplete = false;
			}

			if (i < word2.length())
			{
				result.append(word2.charAt(i));

				isComplete = false;
			}

			if (isComplete)
			{
				break;
			}
		}

		return result.toString();
	}
}

File: solutions/1768/js/solution.js
// 1768. Merge Strings Alternately
var mergeAlternately = function(word1, word2)
{
	let result = "";

	for (let i = 0;; ++i)
	{
		let isComplete = true;

		if (i < word1.length)
		{
			result += word1[i];

			isComplete = false;
		}

		if (i < word2.length)
		{
			result += word2[i];

			isComplete = false;
		}

		if (isComplete)
		{
			break;
		}
	}

	return result;
};